.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Was node_modules missing before?

Reply:
I somehow had node_modules in the top level directory. I removed node_modules locally but it can't hurt to have that in the `.gitignore`.